### ISSUE TITLE
Idea sketch: Error message might be too subtle

### ISSUE THREAD

**Issue by rittermo** [liqd/a4-advocate-europe#64]
I think just changing the text color is a little too subtle for error messages to be recognized as such. Also, I think it is hard to see, which form field the error message belongs to.

![bildschirmfoto 2017-05-11 um 17 51 17](https://cloud.githubusercontent.com/assets/15341015/25958950/7f2522a2-3672-11e7-9038-ceada0096de7.png)

**Comment by rittermo** [comment]
What do you think, @damusp ?

**Comment by damusp** [comment]
Hm, I agree. Thanks for pointing that out! Actually, the alert message is supposed to go right beneath the invalid field.

The standard error style I delivered was this:
![error01](https://cloud.githubusercontent.com/assets/22769111/25959451/eb5ee060-3673-11e7-9fd8-c3c368e79f7c.png)

But maybe we could use something even stronger, like the one I have designed for Opin? In AE, it would result in something like this:
![error02](https://cloud.githubusercontent.com/assets/22769111/25959464/f2feb84a-3673-11e7-85ca-be8f3f108441.png)
